corehq/motech/migrations/0009_partition.py

```diff
@@ -0,0 +1,109 @@
+from django.db import migrations
+from django.db.migrations.operations.models import ModelOperation
+
+from architect.commands import partition
+
+
+def get_sql():
+    next_month = '2021-09-01 00:00:00 Z'  # TODO: Calculate this
+    return f"""
+        CREATE TABLE motech_requestlog (
```

Comment:
if this table has exactly the same schema as the existing one you could say `CREATE TABLE motech_requestlog LIKE dhis2_jsonapilog INCLUDING ALL`

https://www.postgresql.org/docs/9.6/sql-createtable.html